docs/modules/ROOT/pages/getting-started/roles_persmissions.adoc

```diff
@@ -1,20 +1,20 @@
 = Permissions and roles
 
-As a workspace owner, you can https://placeholder.com[invite] one or more collaborators to your workspace by adding their user ID, such as an external SSO ID. After https://placeholder.com[adding a user to a workspace], you assign them a specific role that determines the actions they can take in a workspace.
+As a workspace owner, you can invite collaborators to your workspace by adding their user ID, such as an external SSO ID. After adding a user to a workspace, you assign them a specific role that determines the actions they can take in that workspace.
 
-The roles available for a workspace, from least access to most access, are:
+The roles available for a workspace, from least access to most access, are the following:
 
-* *Contributor*: Ideal for users who actively push to your workspace.
+* *Contributor*: For users who actively push to your workspace.
 
-* *Maintainer*: Ideal for users who need to manage the workspace without access to sensitive actions like managing security or deleting a workspace.
+* *Maintainer*: For users who need to manage the workspace without access to sensitive actions like managing security or deleting a workspace.
```

Comment:
No contributor or admin can delete a workspace. A workspace can be only deleted by RHTAP admins on request.

Reply:
@alexeykazakov is "Owner" an RHTAP admin?

Reply:
No. There is no such a role as "Owner". We might introduce it in the future but we don't have it now.
HAC/UI uses "Owner" but it's a bug. They are working on fixing it and start using "Admin" instead.